llvm/lib/YkIR/YkIRWriter.cpp

```diff
@@ -25,6 +25,9 @@
 using namespace llvm;
 using namespace std;
 
+// For a patchpoint call, the argument index of the number of target arguments.
```

Comment:
Why 3?

Reply:
I think it's explained in the actual code:

```
// Skip the following arguments: ID, shadow, target, target arguments.
```

although that code doesn't seem to make use of this constant?

Reply:
It's used here:
https://github.com/ykjit/ykllvm/pull/205/files/13eb0fe70d25879c5a3572657d4bf4d9f4b390e4#diff-9a98c82d803f659ad8f727223c544d73eab8f1967e132949117784b434d64381R597

The reason it's 3 is that it's the argument index of the `<numArgs>` parameter to the patchpoint intrinsic. We need this later to know where arguments to `__ykrt_control_point` stop, and where live variables to include in the stackmap entry start.

https://llvm.org/docs/StackMaps.html#llvm-experimental-patchpoint-intrinsic

Reply:
OK, let's document that please.

Reply:
Fixed in b9505623d9d4b6777095d44d3beb57745484ea6f